<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calcdetailFix</class>
 <widget class="QWidget" name="calcdetailFix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculation Parts Fix</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>&lt;h1&gt;Fix Cost Parts&lt;/h1&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel2">
     <property name="text">
      <string>Add a fix cost for one unit of the template:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel3">
       <property name="text">
        <string>&amp;Label:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>m_nameEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_nameEdit">
       <property name="toolTip">
        <string>describing text</string>
       </property>
       <property name="text">
        <string>amortisation</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel3_2">
       <property name="text">
        <string>&amp;Amount:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>m_inpMenge</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="m_inpMenge">
       <property name="toolTip">
        <string>amount multiplier</string>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabel3_2_2">
       <property name="text">
        <string>at &amp;Price:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>m_inpPreis</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="m_inpPreis">
       <property name="toolTip">
        <string>Price for one piece</string>
       </property>
       <property name="suffix">
        <string>â‚¬</string>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
       <property name="maxValue" stdset="0">
        <number>999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>m_nameEdit</tabstop>
  <tabstop>m_inpMenge</tabstop>
  <tabstop>m_inpPreis</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
